pkg/controller/controller.go

```diff
@@ -98,8 +98,6 @@ func (c *Controller) CreateCluster(cluster model.Cluster, assets model.Assets) e
 	if cluster.Labels == nil {
 		cluster.Labels = model.Labels{}
 	}
-	// Set default cluster labels.
-	cluster.Labels[constants.ClusterNameLabelKey] = cluster.Name
 
```

Comment:
I figured that `cluster-name` is not really useful. Why does keto-tokens need this label?